front/components/workspace/ProviderManagementModal.tsx

```diff
@@ -37,7 +37,7 @@ const prettyfiedProviderNames: { [key in ModelProviderIdType]: string } = {
 const modelProviders: Record<ModelProviderIdType, string[]> =
   USED_MODEL_CONFIGS.reduce(
     (acc, model) => {
-      if (!model.isLegacy) {
+      if (model && !model.isLegacy) {
```

Comment:
FMI: how come do you need that?

Reply:
For some reasons, I couldn't load the workspace page, I had to add this line to make it work (suggested by @q)

Reply:
can you try again without now that we have `types` running?